Source/WebCore/platform/ScrollableArea.h

```diff
@@ -141,6 +141,8 @@ class ScrollableArea : public CanMakeWeakPtr<ScrollableArea>, public CanMakeChec
     virtual OverscrollBehavior horizontalOverscrollBehavior() const { return OverscrollBehavior::Auto; }
     virtual OverscrollBehavior verticalOverscrollBehavior() const { return OverscrollBehavior::Auto; }
 
+    virtual ScrollbarWidth scrollbarWidth() const { return ScrollbarWidth::Auto; }
```

Comment:
Slightly uneasy with the name (it'd expect it to return an `int`). Maybe `scrollbarWidthStyle()`?